mlflow/catboost.py

```diff
@@ -59,6 +59,8 @@ def save_model(
     mlflow_model=None,
     signature: ModelSignature = None,
     input_example: ModelInputExample = None,
+    pip_requirements=None,
+    extra_pip_requirements=None,
```

Comment:
We still need to provide implementations for each of these, right?

Reply:
Yes, it's still WIP :)